components/datetime/src/provider/time_zones.rs

```diff
@@ -162,7 +162,12 @@ pub struct ExemplarCities<'data> {
 /// </div>
 #[icu_provider::data_struct(
     marker(MetazoneGenericNamesLongV1, "time_zone/generic_long@1", has_checksum),
-    marker(MetazoneGenericNamesShortV1, "time_zone/generic_short@1", has_checksum)
+    marker(MetazoneGenericNamesShortV1, "time_zone/generic_short@1", has_checksum),
+    marker(
+        MetazoneGenericStandardNamesLongV1,
+        "time_zone/generic_standard_long@1",
+        has_checksum
```

Comment:
Suggestion: Maybe name it `metazone/standard/names/long/v1` (on the same level as generic and standard)?